### PR TITLE
Command space suits

### PR DESCRIPTION
**Pull request by Kmc2000**
They're back except NOT shit this time, they are fully working and usable, 

But, Kmc, why would you make these

because it's always good for the heads to have a backup in case mc greyshirt steals their main hardsuit.

PICS OR IT DIDN'T HAPPEN:

Wew lad i did the thing

### CODE REVIEW

**Comment by Nirnael** [comment]
Oh god KMC is touching code, all duck.

**Comment by Kmc2000** [comment]
> fizzle

I want to be the very best like no one ever was

To code them is my test to compile them is my dream

Spritemappercodeadminemon

**Comment by Super3222** [comment]
For everyone that's going going to have a problem with this since each head has their own hardsuit:
1. HoP doesn't have a hardsuit. Bingo.
2. EVA is used for emergencies, and having spare exclusive hardsuits for the command members for the station isn't a bad idea. For instance, when half of the station is eaten by a giant singularity and you can't get to your office.
3. More hardsuits to give out in general is good because people come in droves to E.V.A. during meteor / blob / nukeop who are winning / whatever the situation may be.

**Comment by Nirnael** [comment]
There are a total of 8 suits from each escape pod that open during code red, those are never used.

**Comment by Kmc2000** [comment]
> fizzle again

yeah but heads need more snowflakes in their lives, they run the station

**Comment by GrayRachnid** [comment]
:-1:

**Comment by AsV9** [comment]
why'd you change a total of 126 lines in the map? did you edit some shit which you forgot to mention?

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -37,6 +37,12 @@
 	mask_type = /obj/item/clothing/mask/gas
 	storage_type = /obj/item/weapon/tank/jetpack/oxygen/captain
 
+/obj/machinery/suit_storage_unit/command
+	suit_type = /obj/item/clothing/suit/space/heads
+	helmet_type = /obj/item/clothing/head/helmet/space/heads
+	mask_type = /obj/item/clothing/mask/breath
+
+
```
Comment (AsV9) [inline code/game/machinery/suit_storage_unit.dm:45]: this fucking empty line is driving me nuts.

**Comment by Kmc2000** [comment]
> asv

That'd be the mapmerger i'd imagine totally didnt turn sec into a clown brothel or nufin

**Comment by Super3222** [comment]
That is NOT the map merge. It just means you have to backspace a bit @Kmc2000

**Pull request merged**